eo-maven-plugin/src/test/java/org/eolang/maven/BinarizeMojoTest.java

```diff
@@ -71,7 +71,8 @@ void binarizesWithoutErrors(@TempDir final Path temp) throws Exception {
                 .withProgram(BinarizeMojoTest.SRC.resolve("twice-rust.eo"));
         }
         Assertions.assertDoesNotThrow(
-            () -> maven.execute(new FakeMaven.Binarize())
+            () -> maven.execute(new FakeMaven.Binarize()),
+            "TO ADD ASSERTION MESSAGE"
```

Comment:
@volodya-lombrozo I think it is better to keep repeating message where it is allowed. The advantages of this is that you will not need to think about deleting unused field when meaningful messages will be added. And I don't see any disadvantages.

We already have a check for this, and this check prohibits more than 5 repetitions of the same values (and in such cases I did as you suggested).

Reply:
@c71n93, I haven't clearly understood the "advantages" you described. To me, they actually seem like disadvantages. Anyway.

Please take another look at [this code](https://github.com/objectionary/eo/pull/3113#discussion_r1572239134) and pay attention to the comment. I suggested this change intentionally.

Why we should opt for only one field instead of many different fields and repeated `"TO ADD ASSERTION MESSAGE"` constants:

1. You have only **one** place where you define this message.
2. You have only **one** place where you can add a todo puzzle for further changes.
3. It's the **only** place that has a [bad smell](https://www.yegor256.com/2015/07/06/public-static-literals.html). Otherwise you infecting the codebase by _suboptimal_ code in many places.  


Otherwise, if you leave it as it is:

1. You need to add a separate puzzle for each field.
2. If you leave the "TO ADD ASSERTION MESSAGE" constants, we will just forget to replace them with meaningful messages.
3. You have to deal with checkstyle check:

> this check prohibits more than 5 repetitions of the same values

Reply:
@c71n93 Moreover, in the future versions of `jtcop` we [will forbid the usage of static literals](https://github.com/volodya-lombrozo/jtcop/issues/302). So, later we will need to remove all of this static literals.

Reply:
@volodya-lombrozo sorry, I misunderstood what you were suggesting. Just to clarify: you are suggesting to add such `public static` fields with todo to any test class in `eo-runtime`, `eo-paser`, `eo-maven-plugin` and use this field in other tests. Am I right?

Reply:
@c71n93 Maybe we will add three similar constants for each submodule: 

```java
// todo: remove this field from 'eo-runtime'
public static final  String TODO_MESSAGE = "TO ADD ASSERTION MESSAGE";

// todo: remove this field from 'eo-parser'
public static final  String TODO_MESSAGE = "TO ADD ASSERTION MESSAGE";

// todo: remove this field from 'eo-maven-plugin'
public static final  String TODO_MESSAGE = "TO ADD ASSERTION MESSAGE";
```

In this case we will have 3 separate todos which will be much easier to fix and we don't need to think how to share the same field across modules. What do you think?

Reply:
Yes, it some sort of duplication, but it's [simple](https://en.wikipedia.org/wiki/KISS_principle), as for me.

Reply:
> add such public static fields with todo to any test class

Not exactly, I suggest to leave only one field and use it across all of the other classes.

Reply:
@volodya-lombrozo Ok, understood. I free to choose any class in every submodule?

Reply:
@c71n93 Yes, it doesn't matter which class you will choose, since we will remove it later.